atom/common/native_mate_converters/value_converter.h

```diff
@@ -24,6 +24,12 @@ struct Converter<base::DictionaryValue> {
 };
 
 template<>
+struct Converter<const base::DictionaryValue*> {
```

Comment:
I don't think we need to add a converter for the pointer type, we can just do something like `Emit('blabla', *dict)` in code.